gitops/utils/apps.py

```diff
@@ -13,25 +14,25 @@
 from .images import colour_image
 from .tags import colour_tags, validate_tags
 
+APPS_PATH = Path(os.environ.get('GITOPS_APPS_DIRECTORY', 'apps'))
 
-def get_app_details(app_name, load_secrets=True):
+
+def get_app_details(app_name: str, load_secrets: bool = True) -> App:
     account_id = get_account_id() if load_secrets else 'UNKNOWN'
     try:
-        app = App(app_name, path=f'apps/{app_name}', load_secrets=load_secrets, account_id=account_id)
+        app = App(app_name, path=APPS_PATH / app_name, load_secrets=load_secrets, account_id=account_id)
     except FileNotFoundError:
         # Check if apps dir doesn't exist, or just that one app
         if os.path.exists("apps"):
             raise AppDoesNotExist(app_name)
         else:
             raise AppDoesNotExist()
 
-    values = app.values
-    values['name'] = app_name
-    return values
+    return app
```

Comment:
Yeah this is much nicer. I'm not sure what the reasoning was for dicts.. maybe some serialisation thing? ;/